crates/bevy_ui/src/render/box_shadow.wgsl

```diff
@@ -0,0 +1,95 @@
+#import bevy_render::view::View;
+#import bevy_render::globals::Globals;
+
+const PI: f32 = 3.14159265358979323846;
+
+@group(0) @binding(0) var<uniform> view: View;
+@group(0) @binding(1) var<uniform> globals: Globals;
+
+struct BoxShadowVertexOutput {
+    @builtin(position) position: vec4<f32>,
+    @location(0) point: vec2<f32>,
+    @location(1) color: vec4<f32>,
+    @location(2) @interpolate(flat) size: vec2<f32>,
+    @location(3) @interpolate(flat) radius: vec4<f32>,    
+    @location(4) @interpolate(flat) blur: f32,
+}
+
+fn gaussian(x: f32, sigma: f32) -> f32 {
```

Comment:
I think this would be clearer if `x` was renamed to `mu`, or even better if we swapped to `mean`/`variance`.